{{cookiecutter.project_dirname}}/docker-compose.yaml

```
@@ -46,17 +46,22 @@ services:
       - ./{{ cookiecutter.frontend_service_slug }}:/app{% endif %}
 
   traefik:
-    image: traefik:v2.6
+    command:
+      - "--configFile=/traefik/conf/static.yaml"
```

As you're refactoring it, I'd make it more compact
```suggestion
    command: "--configFile=/traefik/conf/static.yaml"
```